packages/components/src/templates/next/layouts/Collection/Collection.stories.tsx

```diff
@@ -20,7 +20,7 @@ const COLLECTION_ITEMS: IsomerSitemap[] = flatten(
       layout: "article",
       summary:
         "We’ve looked at how people’s spending correlates with how much microscopic plastic they consumed over the year. We’ve looked at how people’s spending correlates with how much microscopic plastic they consumed over the year.",
-      date: "2024-05-07",
+      date: "07/05/2024",
```

Comment:
hmmmm still think ISO Date format is more standard. Can we not use that standard and transform it properly instead?

Reply:
Hmm yeah true, reverted in 3b67c53d9a03540807094d60ce84e34ef8e0d835.